core/src/main/java/io/parsingdata/metal/expression/value/Value.java

```diff
@@ -31,6 +32,7 @@
 
 public class Value {
 
+    public static final Value NOT_A_VALUE = new Value(Slice.createFromBytes(new byte[]{}), DEFAULT_ENCODING);
```

Comment:
Use `new byte[0]` instead.

Reply:
And maybe extract an `EMPTY_SLICE`.

Reply:
As you know, I ended up extracting an entire class :sweat_smile: